client/Combatant/Combatant.ts

```diff
@@ -25,6 +25,7 @@ export interface Combatant {
     StatBlock: KnockoutObservable<StatBlock>;
     GetInitiativeRoll: () => number;
     IsPlayerCharacter: boolean;
+    ImageURL: KnockoutObservable<string>;
```

Comment:
I don't think you need to add this property onto the Combatant, since you can access it directly from the StatBlock where you need it. It's OK to put this property onto StaticCombatantViewModel though.